AppController/djinn.rb

```diff
@@ -2283,7 +2285,7 @@ def receive_server_message(timeout, secret)
         # Client will process "Error" and try again unless appscale is
         # terminated
         if @done_terminating and @waiting_messages.empty?
-          return "Error: Done Terminating and No Messages"
+          raise "Error: Done Terminating and No Messages"
```

Comment:
@cdonati I misinterpreted something you commented on the appscale-tools side then, fixing it.